docsource/functionalRequirements.rst

```diff
@@ -1,2 +1,24 @@
 Functional Requirements
-========================
\ No newline at end of file
+========================
+
+1. Support the indexing of many data types (Genome, Assembly, Narrative etc). It is expected that there will be about 100 types
+
+2. Support the indexing of versions of types. It is expected that each object can have on average about 100 versions.
```

Comment:
I think it should be at most 100 versions, not on average. Based on https://ci.kbase.us/#catalog/datatypes

max = 14
avg = 2.4
stddev = 2.2
n = 254

That's for 4 years of production use, so it's probably not going to change a whole lot.

Also object -> type

Reply:
Changed.